Content/docs/cloudservice/dec update (#6315)
EnterpriseDB/docs: advocacy_docs/edb-postgres-ai/cloud-service/release_notes/index.mdx

Reviewer: Should we add the RN for Nov here?

Reviewer 2: Done

indexCards: none
---

!!!note "Deprecation"

> Please be advised that the EDB Hosted Cloud Service has been deprecated. Support will still be available for current customers. However the related documentation topics removal work is in progress and will be removed shortly. We recommend affected users transition to alternative solutions as soon as possible. Further updates will be provided as the transition progresses.

Reviewer: I think the note is good as it is, just making some suggestions to make it more assertive, but feel free to accept or reject :) 
```suggestion
The EDB Hosted Cloud Service has been deprecated. Support is available for current customers. However, the related documentation topics will be removed shortly. We recommend that affected users transition to an alternative solution immediately. Further updates will be provided as the removal progresses.
``` 
I also think instead of "We recommend that affected users transition to an alternative solution inmediately." we should say something more helpful like: "Our Professional Services team can support your transition into an alternative solution on a case-by-case basis." (But I don't know if support services will actually help, so we'd have to check with PM.
